sound/soc/sof/topology.c

```diff
@@ -15,6 +15,7 @@
 #include <linux/workqueue.h>
 #include <sound/tlv.h>
 #include <uapi/sound/sof/tokens.h>
+#include <sound/sof/topology.h>
```

Comment:
why do you need to include this?

Reply:
Oh its relic from an earlier version. Will remove.